dataprofiler/profilers/utils.py

```diff
@@ -343,3 +343,39 @@ def biased_kurt(df_series):
 
     kurt = n * M4 / M2 ** 2 - 3
     return kurt
+
+def find_diff(stat1, stat2):
```

Comment:
I suggest making individual functions for the differences and allow the creator of the report to set what they want where for each difference.

Reply:
So like allowing the user to overall set chi squared for all number differences or do you mean like the user can set chi squared for specific number differences like if they wanted max diff to be chi squared but still have mean be subtraction?

Reply:
in either case, I'm not sure I want to add options when the options dont exist yet?